Gemfile

```diff
@@ -48,6 +48,9 @@ group :development, :test do
 end
 
 group :development do
+  # GOV.UK interpretation of rubocop for linting Ruby
+  gem 'govuk-lint'
```

Comment:
I think you'll want this in your test group too other travis/vsts can't use it...

Reply:
Good catch, should have been in the development and test group.